### DISCUSSION TITLE
How to use these annotations?

### DISCUSSION THREAD

**Discussion by muhsaad47** [sumatrapdfreader/sumatrapdf#4102] [Q&A]
In the commands section on sumatrapdf site there are these commands that I cant seem to use even after assigning them hotkeys in the settings.txt file. Also they dont show up anywhere in the in app annotation options.
<img width="464" alt="2024-02-23 10_46_14-Commands" src="https://github.com/sumatrapdfreader/sumatrapdf/assets/160825797/427aef8b-f221-4782-802c-fa40476227a9">

**Comment by GitHubRulesOK** [comment]
They should / did pop-up a "NYI" message to say Not Yet Implemented
